### PR TITLE
Enhanced April Fools' Mancrowave + subtype

### PR DESCRIPTION
**Pull request by ghost**
Adds unique icons and additional functionality for the mancrowave when it's April Fools'.
Also adds a unique subtype of the mancrowave that uses the April Fools' behavior all the time.

[sprites]
:cl:
 * imageadd: Enhances the mancrowave's functionality on April Fools', with new sprites by Blithering.

### CODE REVIEW

File: code/game/machinery/Sleeper.dm [hunk 1]
```diff
@@ -661,7 +664,16 @@
 						playsound(src, 'sound/effects/pop.ogg', 50, 1)
 						H.my_appearance.h_style = "Popped Hair"
 						H.update_hair()
-				else if(isjusthuman(H) && Holiday == APRIL_FOOLS_DAY)
+				else if(isjusthuman(H) && (Holiday == APRIL_FOOLS_DAY || istype(src, /obj/machinery/sleeper/mancrowave/galo)))
```
Comment (jknpj) [inline code/game/machinery/Sleeper.dm]: Wouldn't a var be better for this?

Comment (ghost) [reply code/game/machinery/Sleeper.dm]: I wasn't sure what the ideal way to do it was but I changed it to a var.

File: code/game/machinery/Sleeper.dm [hunk 2]
```diff
@@ -661,7 +665,17 @@
 						playsound(src, 'sound/effects/pop.ogg', 50, 1)
 						H.my_appearance.h_style = "Popped Hair"
 						H.update_hair()
-				else if(isjusthuman(H) && Holiday == APRIL_FOOLS_DAY)
+				else if(isjusthuman(H) && (Holiday == APRIL_FOOLS_DAY || galize == 1))
```
Comment (jknpj) [inline code/game/machinery/Sleeper.dm]: just galize is fine, since it's a binary var

File: code/game/machinery/Sleeper.dm [hunk 3]
```diff
@@ -505,10 +505,14 @@
 	automatic = TRUE
 	drag_delay = 0
 	machine_flags = SCREWTOGGLE | CROWDESTROY | EMAGGABLE | EJECTNOTDEL
+	var/galize = 0
 
 /obj/machinery/sleeper/mancrowave/New()
 	..()
-	if(map.nameShort == "deff")
+	if(Holiday == APRIL_FOOLS_DAY)
+		base_icon = "galo"
```
Comment (jknpj) [inline code/game/machinery/Sleeper.dm:513]: could make it turn the galize var on

File: code/game/machinery/Sleeper.dm (hunk 2 above)
Comment (jknpj) [reply code/game/machinery/Sleeper.dm]: if you make the holiday check from before turn the galize var on, you could remove this holiday check

**Pull request merged**